Add language and labels to the activities index

winniederidder opened the pull request:
This pull request updates the activity listings.

For comparison: the upper image is the original series edit. Below is the new activities index page with now more data added.
![series-edit](https://user-images.githubusercontent.com/53743234/90114584-346ffd00-dd53-11ea-8386-47db7a4451d9.PNG)
![activities-index](https://user-images.githubusercontent.com/53743234/90114589-36d25700-dd53-11ea-8cc2-5cde1b04c585.PNG)

The labels, repository names etc. are clickable. For this I created an activities pack and moved initTokensClickable from series to util as it is no longer series only.

Closes #2163  .

bmesuere reviewed (commented):
I would remove the "labels" (and potentially "languages") header since the column contains both the programming language and the labels.

bmesuere commented on app/javascript/packs/activities.js:
```diff
@@ -0,0 +1,3 @@
+import { initTokenClickables } from "util.js";
+
+window.dodona.initTokenClickables = initTokenClickables;
```
Is there a reason to bind this to the global dodona variable?

winniederidder replied:
I did what other packs did. I'll see for a different way.

bmesuere commented:
The activities index view should be set to use 12 columns instead of 10 right now.

winniederidder commented:
![activities-index](https://user-images.githubusercontent.com/53743234/91040504-a62b2f00-e60e-11ea-8014-94716a98b98d.PNG)
Changed to 12 columns and removed labels header

bmesuere commented on app/views/activities/index.html.erb:
```diff
@@ -1,5 +1,5 @@
 <div class="row">
-  <div class="col-sm-10 col-sm-offset-1 col-xs-12">
+  <div class="col-sm-12 col-sm-offset-1 col-xs-12">
```
```suggestion
  <div class="col-xs-12">
```
